shell/platform/darwin/macos/framework/Source/FlutterTextInputSemanticsObject.h

```diff
@@ -69,24 +69,16 @@ class FlutterTextPlatformNode : public ui::AXPlatformNodeBase {
  * This causes VoiceOver to announce text editing feedbacks.
  *
  * All of the text editing events in this native text field are redirected to the
- * FlutterTextInputPlugin and will eventually update this text field through text input channel.
+ * text input plugin returned by the FlutterTextFieldDelegate.
  */
 @interface FlutterTextField : NSTextField
 
 /**
- * Initializes a FlutterTextField.
- *
- * The plugin will be stored as an __unsafe_unretained pointer, and the FlutterTextField expects
- * the point is valid during its lifetime.
- */
-- (instancetype)initWithPlatformNode:(flutter::FlutterTextPlatformNode*)node;
-
-/**
- * Assign the field editor for this FlutterTextField. The plugin will be stored as an
- * __unsafe_unretained pointer, and the FlutterTextField expects the pointer is valid
- * during its lifetime.
+ * Initializes a FlutterTextField that uses the textInputPlugin from the FlutterTextFieldDelegate
+ * as its field editor.
  */
-- (void)setFieldEditor:(FlutterTextInputPlugin*)plugin;
+- (instancetype)initWithPlatformNode:(flutter::FlutterTextPlatformNode*)node
+                           andEngine:(FlutterEngine*)engine;
```

Comment:
Do not use "and" for additional arguments, per [ObjC style](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html)